playbooks/ironic-cimc-job-pre.yaml

```diff
@@ -1,8 +1,20 @@
 ---
+# FIXME(sambetts) this should become part of the image build process
 - hosts: all
+  tasks:
+    - file:
+        path: /home/jenkins
+        mode: 0755
+
+- hosts: all
+  vars:
+    additional_pip_libraries:
+      - "ImcSdk==0.7.2"
```

Comment:
Is this potentially tied to the driver version?  Crossing that bridge when it comes is fine, IMO.  Just curious whether it's a dependency tied across release, ironic driver type/version, and claimed baremetal node CIMC version.

Reply:
Yeah it's definitely tied to the driver version being tested. My SDK update changes that dependency so we should add some logic to determine version here, we can do that in a follow up because everything that is currently out is using 0.7.2